### PR TITLE
Migrate RBMoveMethodToClassSide to refactoring2

### PR DESCRIPTION
**Pull request by miguelcamp**
Also includes some small changes to RBAddVariableAccesorTransformation for use in MoveMethodToClassSide

### CODE REVIEW

File: src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	(class whoDefinesClassVariable: aString) ifNotNil: 
+			[^ true].
+	^ (self parseTree allDefinedVariables includes: aString) 
+]
+
+{ #category : #refactoring }
+RBMoveMethodToClassSideTransformation >> getNewInstSideSource [
+	| sender |
+	sender := ''.
+	(method argumentNames size > 0) ifTrue: [
```
Comment (Ducasse) [inline src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st]: No need for parentheses

File: src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+			[^ true].
+	^ (self parseTree allDefinedVariables includes: aString) 
+]
+
+{ #category : #refactoring }
+RBMoveMethodToClassSideTransformation >> getNewInstSideSource [
+	| sender |
+	sender := ''.
+	(method argumentNames size > 0) ifTrue: [ 
+		(method selector keywords) with: (method argumentNames )do: [:a :b |
```
Comment (Ducasse) [inline src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st]: no need for parantheses

File: src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+]
+
+{ #category : #refactoring }
+RBMoveMethodToClassSideTransformation >> getTempName [
+	| aString counter tempName |
+	counter := 0.
+	aString := class canonicalArgumentName.
+	tempName := aString.
+	[self checkVariableNamed: tempName]
+	whileTrue: [ counter := counter + 1.
```
Comment (Ducasse) [inline src/Refactoring2-Transformations/RBMoveMethodToClassSideTransformation.class.st]: tab before whileTrue please
And getTempName could be just temporaryName

**Comment by MarcusDenker** [comment]
Now there is a conflict

**Pull request merged**